### PR TITLE
Impove the way we checkForChildTooFarAway in nearParentGrabEntity.js

### PR DESCRIPTION
**Pull request by danteruiz**
Revert back the way we checkForChildTooFarAway in nearParentGrabEntity.js

- https://highfidelity.fogbugz.com/f/cases/16974/Near-Grab-Jitters-along-its-edge-boundaries

### CODE REVIEW

**Comment by danteruiz** [comment]
Test plan
1) Enter a domain with create permissions and create a cube.
2) Reach toward the cube and grab it along the outside of Near-Grab's radius.
3) Walk backwards while still holding grab, and the cube should not be left behind.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11604/HighFidelity-Beta-PR13700-74bb230.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11604/HighFidelity-Beta-PR13700-74bb230.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11604/HighFidelity-Beta-Interface-PR13700-74bb230.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11604/HighFidelity-Beta-PR13700-74bb230.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11604/HighFidelity-Beta-Interface-PR13700-74bb230.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11640/HighFidelity-Beta-PR13700-cf6850d.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11640/HighFidelity-Beta-PR13700-cf6850d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11640/HighFidelity-Beta-Interface-PR13700-cf6850d.exe

File: scripts/system/controllers/controllerModules/nearParentGrabEntity.js [hunk 1]
```diff
@@ -23,6 +23,20 @@ Script.include("/~/system/libraries/cloneEntityUtils.js");
     // XXX this.ignoreIK = (grabbableData.ignoreIK !== undefined) ? grabbableData.ignoreIK : true;
     // XXX this.kinematicGrab = (grabbableData.kinematic !== undefined) ? grabbableData.kinematic : NEAR_GRABBING_KINEMATIC;
 
+    var GRAB_POINT_SPHERE_OFFSET = { x: 0.01, y: -0.13, z: 0.039 };
```
Comment (sethalves) [inline scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: could we include scripts/system/libraries/controllers.js for this value?  actually... they don't match.  what's up with that?

Comment (danteruiz) [reply scripts/system/controllers/controllerModules/nearParentGrabEntity.js]: They are the same values but in different coordinate systems, I could rename the variable if you wish

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11666/HighFidelity-Beta-PR13700-0f46001.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11666/HighFidelity-Beta-PR13700-0f46001.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11666/HighFidelity-Beta-Interface-PR13700-0f46001.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11666/HighFidelity-Beta-PR13700-0f46001.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11666/HighFidelity-Beta-Interface-PR13700-0f46001.exe

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Comment by qliejahner** [comment]
PASS @conklin94122 

Objects no longer shudder or fall away when near grabbed at the edge.

**Pull request merged**